contracts/01-tamagotchi/io/src/lib.rs

```diff
@@ -10,29 +10,45 @@ use scale_info::TypeInfo;
 #[scale_info(crate = gstd::scale_info)]
 pub struct Tamagotchi {
     // TODO: 1️⃣ Add `name` and `age` fields
+    pub name: String,
+    pub date_of_birth: u64,
 }
 
 #[derive(Encode, Decode, TypeInfo)]
 #[codec(crate = gstd::codec)]
 #[scale_info(crate = gstd::scale_info)]
 pub enum TmgAction {
     // TODO: 2️⃣ Add `Name` and `Age` actions that set the name and age
+    Name,
+    Age,
 }
 
+#[derive(Encode, Decode, TypeInfo)]
+#[codec(crate = gstd::codec)]
+#[scale_info(crate = gstd::scale_info)]
+pub struct  TmgInit {
```

Comment:
This structure seems to not be used.

Reply:
Thank you.  I learning GitHub and Gear protocol. Thank you for your help.